kolibri/plugins/facility/assets/src/views/users/sidePanels/EnrollLearnersSidePanel.vue

```
@@ -209,7 +217,8 @@
             const newMemberships = await MembershipResource.saveCollection({ data: enrollments });
             createdMemberships.value = newMemberships;
           } catch (error) {
-            showErrorWarning.value = true;
+            // Why doesn't this ever get run here?
```

I don't know what this comment means?

---

Thanks for catching this lol - it is an artifact of my confusion caused by the errors being swallowed up in the apiResource module :sweat_smile: